### PR TITLE
chore: rename route from api-v2 to api

### PR DESCRIPTION
**Pull request by riqwan**

### CODE REVIEW

File: www/utils/packages/docblock-generator/src/classes/kinds/oas.ts [hunk 1]
```diff
@@ -632,11 +632,8 @@ class OasKindGenerator extends FunctionKindGenerator {
...
   } {
     const filePath = node.getSourceFile().fileName
-    const oasPath = (
-      filePath.includes("/api-v2/")
-        ? filePath.substring(filePath.indexOf("/api-v2/"))
-        : filePath.substring(filePath.indexOf("/api/"))
-    )
+    const oasPath = filePath
+      .substring(filePath.indexOf("/api/"))
       .replace(/^\/api(-v2)?\//, "")
```
Comment (adrien2p) [inline www/utils/packages/docblock-generator/src/classes/kinds/oas.ts]: suggestion: we can probably remove the v2 notion entirely?

**Review by adrien2p — APPROVED** [review]

File: www/utils/packages/docblock-generator/src/classes/kinds/oas.ts (hunk 1 above)
Comment (riqwan) [reply www/utils/packages/docblock-generator/src/classes/kinds/oas.ts]: good catch, removed!

**Review by shahednasser — APPROVED** [review]
Docs LGTM

**Review by sradevski — APPROVED** [review]
LGTM, together with Virk's PR it will be a nice cleanup 🥳

**Pull request merged**